src/utils/string.js

```diff
@@ -165,3 +166,37 @@ export function getUuid() {
 export function getSwgTransactionId() {
   return getUuid() + '.swg';
 }
+
+/**
+ * Returns a string who's length matches the length of format.
+ * @param {string} str
+ * @param {string} format
+ * @return {string}
+ */
+function padString(str, format) {
+  return (format + str).slice(-format.length);
+}
+
+const PADDING = '00000000';
+function toHex(buffer) {
+  const hexCodes = [];
+  const view = new DataView(buffer);
+  for (let i = 0; i < view.byteLength; i += 4) {
+    // toString(16) will give the hex representation of the number without padding
+    const stringValue = view.getUint32(i).toString(16);
+    hexCodes.push(padString(stringValue, PADDING));
+  }
+  return hexCodes.join('');
+}
+
+/**
+ * @param {string} stringToHash
```

Comment:
Please add a comment for this method

Reply:
To be honest, I'm not sure what a good comment is for this function.

Reply:
Could you describe what it does, and what if might be used for?